velox/exec/Spiller.cpp

```diff
@@ -421,13 +429,23 @@ void Spiller::ensureSorted(SpillRun& run) {
   uint64_t sortTimeNs{0};
   {
     NanosecondTimer timer(&sortTimeNs);
-    gfx::timsort(
-        run.rows.begin(),
-        run.rows.end(),
-        [&](const char* left, const char* right) {
-          return container_->compareRows(
-                     left, right, state_.sortCompareFlags()) < 0;
-        });
+    const auto prefixSortRequiredBytes = PrefixSort::maxRequiredBytes(
+        container_,
+        state_.sortCompareFlags(),
+        state_.prefixSortConfig(),
+        memory::spillMemoryPool());
+    if (memory::spillMemoryPool()->freeBytes() < prefixSortRequiredBytes ||
```

Comment:
spillMemoryPool doesn't have memory limit so we don't need this check here. But I am wondering if we could have a separate config other than query config to enable prefix sort for spill. Thanks!